src/karmen_backend/server/database/printjobs.py

```diff
@@ -48,19 +50,35 @@ def get_printjob(uuid):
         return data
 
 
-def add_printjob(**kwargs):
+def add_printjob(**job):
+    'creates a printjob returning the id of the newly created job'
+
+    if 'uuid' in job:
+        raise ValueError("Cannot add a printjob with an already set UUID.")
+
+    printer = get_printer(job["printer_uuid"])
+    if not printer:
+        raise ValueError(f"The prinetr {job['printer_uuid']} does not exist.")
```

Comment:
Typo prinetr -> printer

Reply:
fixed in 3d2fdbe9ea07f6891d859e0b6a26ca7be9d70139